### DISCUSSION TITLE
Windows DNS logs

### DISCUSSION THREAD

**Discussion by random2342** [utmstack/UTMStack#904] [Q&A] [answered]
Can Windows DNS log files be integrated with UTMStack? I want to correlate firewall events with DNS lookups.
Thanks

**Comment by c3s4rfred** [comment, marked as answer]
Hi @random2342, we have a winlogbeat integration that can manage many Windows events, but if you want to correlate a specific kind of windows logs, you can use "syslog" or "json" integration if you have a way to redirect the logs according to the selected integration steps.

Best regards

**Comment by agauttam** [comment]
The simplest way to fulfill your requirement is by using the Winbeat agent. You can configure Sysmon on your systems to push all events to UTMStack. However, I'm not certain how Sysmon behaves on a Windows DNS server—you can try it, and it may forward all logs related to Windows DNS server events.
